Gruntfile.js

```diff
@@ -1,63 +1,187 @@
-'use strict';
+var request = require('superagent');
 
 module.exports = function(grunt) {
+  'use strict';
+
+  require('load-grunt-tasks')(grunt);
+
   grunt.initConfig({
-    'git-describe': {
-      options: {
+    pkg: grunt.file.readJSON('package.json'),
+    changelog: 'CHANGELOG.md',
+
+    'gh-release': {},
 
-      },
-      'run': {}
-    },
     closurecompiler: {
       minify: {
         requiresConfig: 'git-revision',
         files: {
-          "gen/axs_testing.js": [
```

Comment:
@alice do you have an opinion on using single vs double quotes? We might want to canonize this in a styleguide.